src/app/theme/ThemeProvider.tsx

```diff
@@ -72,6 +72,10 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({ children }) => {
 
   const toggleTheme = (theme: Theme) => setCurrentTheme(theme);
 
+  const persistIsDark = (value: boolean) => {
```

Comment:
nit: the name `persistIsDark` is a bit confusing; you need to know what isDark is. `persistDarkTheme` makes it clearer what it does: persist the dark theme.